### PR TITLE
Fix for trufflesuite/ganache-cli#354 

### PR DESCRIPTION
**Pull request by yarrumretep**
evm_mine returns true in the result to ensure valid JSON_RPC response.

### CODE REVIEW

**Comment by yarrumretep** [comment]
@benjamincburns - ok - updated per your direction.

**Comment by yarrumretep** [comment]
hrm - travis failed, but i think the tests all pass with both of the latest commits?

**Comment by benjamincburns** [comment]
I’ll have a look today. If it just needs a quick fix, I’ll handle it,
provided you ticked the box that lets project members push changes. Thanks
for your contribution!

**Comment by yarrumretep** [comment]
I ticked "allow edits from maintainers" - assume that's what you are referring to.  

It's really a minor contribution - but happy to have my feet wet - hopefully I can contribute more meaningfully in the future!

Rock on!

File: test/mining.js [hunk 1]
```diff
@@ -111,9 +111,10 @@ describe("Mining", function() {
         jsonrpc: "2.0",
         method: "evm_mine",
         id: new Date().getTime()
-      }, function(err) {
+      }, function(err, result) {
         if (err) return reject(err);
-        accept();
+        if(result.result !== '0x0') return reject(new Error("evm_mine should return result = '0x0':" + JSON.stringify(result)))
```
Comment (benjamincburns) [inline test/mining.js]: This works fine, but it's more idiomatic to write `assert.notDeepEqual(result.result, '0.0')`

Comment (benjamincburns) [reply test/mining.js]: No need to resubmit - I'll change it when I merge

**Comment by benjamincburns** [comment]
Looks like the test failure was an unrelated test timing out, which seems to happen on the node lte/boron release from time to time. I'm rerunning that build and provided it doesn't fail a second time around, I'll merge.

**Comment by benjamincburns** [comment]
rebased changes on current develop and pushed back to upstream repo prior to merging

**Comment by benjamincburns** [comment]
Testing snob protip: when one corrects a contributor's nonidiomatic assertion logic, one looks rather foolish when the "correction" breaks the contributor's test. :joy:

Fixed ;-)

**Pull request merged**